python/mxnet/callback.py

```diff
@@ -85,6 +85,7 @@ def __call__(self, param):
         if self.init:
             if count % self.frequent == 0:
                 speed = self.frequent * self.batch_size / (time.time() - self.tic)
+                sys.stdout.write('\n')
```

Comment:
Using logging for a new line will start to print with the logging header, then new line which is not necessary and mess up the printout, and the original printout has such issues.

As in train_model.py, the logging header is like:
`    head = '%(asctime)-15s Node[' + str(kv.rank) + '] %(message)s'
`


This is also consistent with the progressbar printout:
`sys.stdout.write('[%s] %s%s\r' % (prog_bar, percents, '%'))`